src/transport_performance/gtfs/validation.py

```diff
@@ -212,6 +216,12 @@ class GtfsInstance:
         Returns the `daily_route_summary` attribute.
     html_report()
         Generate a HTML report describing the GTFS data.
+    save()
+        Save the current GtfsInstance().
+    filter_to_date()
+        Filter a GtfsInstance to a specific dates or set of dates.
+    filter_to_bbox()
+        Crop a GtfsInstance to a given bbox.
```

Comment:
Worth discussing why GtfsInstance would need to separate out the bbox / date filtering logic as separate methods, unsure if I'm missing something about how this needs to operate.